trulens_eval/examples/expositional/end2end_apps/custom_app/custom_example.ipynb

```diff
@@ -4,9 +4,14 @@
    "cell_type": "markdown",
```

Comment:
<p>I recall we made a decision at some point to have users only use the context manager approach rather than the direct app call (found a reference to it in this closed PR: <a href="https://github.com/truera/trulens/pull/442" target="_blank">https://github.com/truera/trulens/pull/442</a>). Is that valid? If so, can we change this to use the context manager approach?</p><p> </p><p>If so, we should also think about deprecating the <code>with_record</code> and similar methods.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>